src/paperoni/config.py

```diff
@@ -28,10 +28,40 @@ class PaperoniConfig:
     api_keys: Keys[str, str] = field(default_factory=Keys)
     fetch: TaggedSubclass[Fetcher] = field(default_factory=RequestsFetcher)
     focuses: Focuses = field(default_factory=Focuses)
+    autofocus: AutoFocus[str, AutoFocus.Type] = field(default_factory=AutoFocus)
     refine: Refine = None
     work_file: Path = None
     collection: TaggedSubclass[PaperCollection] = None
```

Comment:
I'm not sure why this is not making `MongoCollection` available in the configuration file. It seams it cannot find the definition of the class: https://github.com/mila-iqia/paperoni/actions/runs/17849937365/job/50756415653?pr=87#step:9:374